components/Header.js

```diff
@@ -24,42 +24,58 @@ export default class Header extends Component {
     }
 
     render() {
-      const { header, title, des } = this.props
+      const { header, title, des, intro } = this.props
       if (header) {
         return (
-          <View style={{backgroundColor: 'white'}}>    
-            <Text style={s.header}>{title}</Text>
-            <Text style={s.description}>{des}</Text>
+          <View style={{backgroundColor: 'white'}}>
+            <View style={s.headerBox}>
+              {intro.length ? <Text style={s.header}>{intro}</Text> : null}  
```

Comment:
are `intro`, `title`, etc. guaranteed to not be null?

Since empty strings and null/undefined are all falsy, this would work as well:
`intro ? <Text style={s.header}>{intro}</Text> : null`